script/Arcueid/Day 6/06_00_ARC06_1_1.txt

```diff
@@ -52,7 +52,7 @@ It's strange, but I feel as though it's been quite some time since I last saw yo
 [sha:71b6c0d427aa297be708b47651c60bcec04db507]{
 -- Page 54, Offset 14051.
 -- いや、その……うちの妹、ちょっと迫力ありすぎない？
-Um... Well, isn't she coming in a little hot?
+Um... Is it me, or is she coming in a little hot?
```

Comment:
Feels like it doesn't quite capture what's going on
>Man, is it just me, or is Akiha way too intimidating?

Well, that's how I'd phrase it